backend/test_app/test_account_endpoints.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+from fastapi.testclient import TestClient
+from main import app
+import json
+from domain.user.user_crud import get_user_by_username, get_user_by_id
+from domain.account.account_crud import get_account_by_id
+from database import SessionLocal
+
```

Comment:
validator will throw an error because it will try to strip() current_balance, which is a float. You can remove current_balance from validator.